src/Cardano/TextEnvelope.purs

```diff
@@ -0,0 +1,100 @@
+module Cardano.TextEnvelope
+  ( decodeTextEnvelope
+  , printTextEnvelopeDecodeError
+  , textEnvelopeBytes
+  , TextEnvelope(TextEnvelope)
+  , TextEnvelopeType
+      ( PlutusScriptV1
+      , PaymentSigningKeyShelley_ed25519
+      , StakeSigningKeyShelley_ed25519
+      )
+  , TextEnvelopeDecodeError(JsonDecodeError, CborParseError)
+  ) where
+
+import Prelude
+
+import Aeson
+  ( class DecodeAeson
+  , Aeson
+  , JsonDecodeError(TypeMismatch)
+  , decodeAeson
+  , parseJsonStringToAeson
+  )
+import Control.Monad.Except (throwError)
+import Control.Monad.Error.Class (liftEither)
+import Data.Argonaut (printJsonDecodeError)
```

Comment:
We should probably export this from some `Aeson` modules, I'll open an issue on the repo

Reply:
We don't need to hold up this PR for that, that's more of a note

Reply:
#742 has been merged so we can fix this now